### DISCUSSION TITLE
Download list created by Ai from user promt.

### DISCUSSION THREAD

**Discussion by mastnacek** [KRTirtho/spotube#2191] [Ideas]
Hello man. Im working on python script and gui. I have beta and works great. If you use right prompt and formated output, you obtain search strings for searching. Forexample. Give me top hardest metal songs from 90s. If you interested see how it works, reply.

**Comment by meatmobile** [comment]
lol. nobody wants your scams here, just like nobody wanted it on Newpipe. feel free to attempt some broken czech-to-english insults through google translate though, i dont mind, its really funny

**Reply by mastnacek** [reply]
You are alone, boy. Once again, WHO are you? Dev or nobody? What new feature you have to improve newpipe? New colors?

**Reply by meatmobile** [reply]
Lol

**Reply by mastnacek** [reply]
Lol? So childisch colordeveloper? Empty days in empty live? Go ahead.

**Reply by meatmobile** [reply]
Childish*

Lol
